setup/test_examples/xctest.m

```diff
@@ -0,0 +1,24 @@
+@interface Tests : XCTestCase
```

Comment:
Ok I'll move it into the template, any other comments guys or are you happy to merge otherwise?